pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: lints
-version: 2.0.1
+version: 2.1.0
 description: >
   Official Dart lint rules. Defines the 'core' and 'recommended' set of lints
   suggested by the Dart team.
 repository: https://github.com/dart-lang/lints
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: ^3.0.0
```

Comment:
I think this should be `sdk: ^3.0.0-417` so that we can test it with beta. That should be OK as this is a feature stable beta.

Reply:
cc @natebosch for thoughts too

Reply:
I don't think it makes much difference here in terms of testing with the beta. We should mainly be concerned with whether folks on older SDKs would prefer _not_ to resolve to this version.

I'd expect we would be in favor of projects moving to this set of lints even before they are using the new SDK - am I correct in that assumption?

Reply:
Yes, my point was that folks should be able to move to this using the beta-channel SDK

Reply:
> my point was that folks should be able to move to this using the beta-channel SDK

Do you mean they should _only_ be able to move to this with the beta-channel SDK? In that case I agree, `^3.0.0-417` is a good constraint.

Reply:
Ok, cool. I went ahead and bumped to `^3.0.0-417`.  If that looks good, I'll land and publish.